covalent_ui/webapp/src/components/dispatches/ResultListing.js

```diff
@@ -907,11 +907,15 @@ const ResultListing = () => {
                             />
                           </TableCell>
                         ) : (
-                          <TableCell>{secondsToHms(result.runTime)}</TableCell>
+                          <TableCell>{result.runTime === null ? '-' : secondsToHms(result.runTime)}</TableCell>
```

Comment:
Minor comment that we should check for the falsy value of `result.runTime` to ensure `secondsToHms` doesn't receive falsy values

```javascript
{ !result.runTime ? '-' : secondsToHms(result.runTime)}
```

This can be addressed along with the exception handling changes on a follow up PR

Reply:
I just made the change here so we can disregard above 👍🏼